js-conditions-loops/conditionsAndloops.js

```diff
@@ -0,0 +1,168 @@
+// 1. Нарахуйте зарплату людині яка працює у компанії. Створіть змінну yearsOfExperience яка буде символізувати час, скільки людина працює у компанії.
+// За допомогою конструкції if else, визначте заробітню плату та бонус.
+//Якщо людина працює пів року, то зарплата = 0, бонусів немає
+//Якщо людина працює від пів року до року, то зарплата = 100 , бонус 1 % від зарплати
+//Якщо людина працює від року до двох років, то зарплата = 333, бонус 3 % від зарплати 
+//Якщо людина працює від двох до чотирьох років, то зарплата = 555, бонус 4 % від зарплати 
+//Якщо людина працює від більше чотиьох років, то зарплата = 666 , бонус = ксть років роботи * зарплату
+//Усі результати виведіть у консоль.
+
+const yearsOfExperience = 4;
+
+if (yearsOfExperience <= 0.5) {
+    const salary = 0;
+    const percentage = 0;
+    const bonus = salary / 100 * percentage;
+
+    console.log(`Salary: ${salary}, bonus: ${bonus}`);
+
+} else if (yearsOfExperience <= 1) {
+    const salary = 100;
+    const percentage = 1;
+    const bonus = salary / 100 * percentage;
+
+    console.log(`Salary: ${salary}, bonus: ${bonus}`);
+
+} else if (yearsOfExperience <= 2) {
+    const salary = 333;
+    const percentage = 3;
+    const bonus = salary / 100 * percentage;
+
+    console.log(`Salary: ${salary}, bonus: ${bonus}`);
+
+} else if (yearsOfExperience <= 4) {
+    const salary = 555;
+    const percentage = 4;
+    const bonus = salary / 100 * percentage;
+
+    console.log(`Salary: ${salary}, bonus: ${bonus}`);
+
+} else if (yearsOfExperience > 4) {
+    const salary = 666;
+    const bonus = yearsOfExperience * salary;
+
+    console.log(`Salary: ${salary}, bonus: ${bonus}`);
+
+} else {
+
+    console.log("can not calculate")
+}
```

Comment:
Pls, rework this part. the task is not finished. 
f.e if your yearsOfExperience = 2.5 its calculated inside yearsOfExperience <= 4 block